docs/requirements.md

```diff
@@ -1,16 +1,59 @@
-# PerfCheck CI Requirements
+# Perf Check CI Requirements
 
 Perf Check CI is a web application to manage jobs that measure and compare performance on another web application.
 
 ## Creating a job
 
-* As a user I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I want to compare relative performance of my branch against master to make sure I'm not shipping code that will slow things down.
+* As a user, I want to have a quick idea how performance on my branch looks, without caring about comparisons against master.
+* As a user, I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I know the request I'm working on takes a long time (10-60 seconds), so just want a very rough comparison that takes a smaller amount of time to complete so I can get back to work.
+* As a user, I care about getting a final and accurate-as-possible comparison across two branches.
+* As a user, I want verify that my performance improvements on a branch did not change the response body of the branch.
+
+## Reporting
+
+* As a user, I want to know if the branch performance of each URL is better, about the same, or worse than the reference measurement.
+* As a user, I want to understand how the branch performance thresholds for better/worse are calculated and be able to verify and double check the calculation.
+* As a user, I want to know if URL(s) on my branch still have problematic performance in an absolute way, such as too many database queries, are too long for an acceptable web request, or eats up too much memory.
+
+## Workflow & CI
+
+* As a user, I want to easily start a job from a Github PR without many steps beyond knowing which URL(s) to test.
+* As a user, I want to have a small set of URLs that can be tested against master on most (but not all) of my team's PRs.
+* As a code reviewer, I want to see iterative history that performance was later improved on a PR that initially had poor performance.
+
+## Social Proof
+
+* As a user, I want my GitHub PR to display an up-to-date concise summary of performance status that links to details of the latest Perf Check job for the branch.
+* As a user, I'd like to share my results summary with someone in slack.
+
+## Error Handling
+
+* As a user, I want to know if everything on Perf Check CI is actually working right now.
```

Comment:
I think it makes sense to include some motivation here: *so I can verify that the service is operating properly when my Checks are failing*.